tools/cassandra/README.md

```diff
@@ -9,7 +9,7 @@ make install-schema
 ## For production
 
 ### Create the binaries
-- Run `make bins`
+- Run `make tools`
```

Comment:
Shall we only build the specific tool here instead of building all three tools? Same question for cli and sql README.

Reply:
Yeah that works too. I thought we might want to run a simpler command like `make bins` here.

Reply:
updated :D and add a tip for use `make tools` to build all tools.